pkg/dao/repositories_test.go

```diff
@@ -106,6 +106,7 @@ func (s *RepositorySuite) TestUpdateRepository() {
 		LastIntrospectionUpdateTime:  &expectedTimestamp,
 		LastIntrospectionError:       pointy.String("expected error"),
 		Status:                       config.StatusUnavailable,
+		PackageCount:                 123,
```

Comment:
I would update the `repoPublicTest` and `repoPrivateTest` template variables at `pkg/dao/suite_test.go` to add `PackageCount` field (using values such as `987` and `543`), that should arise a few points to update tests at:

- TestFetchForUrl (Here I have realized a previous confusion usage of `s.repo` and `s.repoPrivate` not related with this change; should I add a separate ticket? it is when comparing the `s.repoPrivate` fetched which is using the `s.repo` instead).
- TestList
- TestUpdateRepository

Reply:
updated, and went ahead and fixed that mixed up confusion on the repo/privateRepo in TestFetchForUrl